plugin/storage/badger/options.go

```diff
@@ -133,11 +132,6 @@ func addFlags(flagSet *flag.FlagSet, nsConfig NamespaceConfig) {
 		nsConfig.MetricsUpdateInterval,
 		"How often the badger metrics are collected by Jaeger. Format is time.Duration (https://golang.org/pkg/time/#Duration)",
 	)
-	flagSet.Bool(
-		nsConfig.namespace+suffixTruncate,
-		nsConfig.Truncate,
-		"If write-ahead-log should be truncated on restart. this will cause data loss.",
```

Comment:
Let's Follow our deprecation policy, keep this flag and log a warning when it is used